README.md

```diff
@@ -3,20 +3,42 @@
 > Instantiate ffmpeg in looping mode.
 
 ## Purpose
-- What problem does this module solve? At least a few sentences.
-PLEASE_FILL_IN_HERE
+Uses fluent-ffmpeg to instantiate an ffmpeg proc in looping mode
 
 ## Usage
 
 ```js
-// Several examples of usage.
-// Usually copying and pasting code from the tests and making the code standalone suffices.
-// PLEASE_FILL_IN_HERE
+const expect = require('expect.js');
```

Comment:
`expect.js`? maybe use `chai` since that is what we use everywhere else these days?